phases/ephemeral/witx/typenames.witx

```diff
@@ -248,8 +248,10 @@
     $path_rename_target
     ;;; The right to invoke `path_filestat_get`.
     $path_filestat_get
-    ;;; The right to change a file's size (there is no `path_filestat_set_size`).
+    ;;; The right to change a file's size.
     ;;; If `path_open` is set, includes the right to invoke `path_open` with `oflags::trunc`.
+    ;;; Note: there is no function named `path_filestat_set_size`. This follows POSIX design,
+    ;;; which only has `ftruncate` and does not provide `ftruncateat`.
```

Comment:
I see.  This makes more sense to me now.

Perhaps you could rephrase.  This comment makes it sound like we don't have this function because we are following POSIX, which is not that case.  How about something like this:

```
Note: there is no function named `path_filestat_set_size` because on many existing
systems there is no way to efficiently implement this.  For example, POSIX has no
`ftruncateat` function.
```

Reply:
Sorry for the delay. I rephrased it in a slightly different way, is it okay for you now?